src/cluster.h

```diff
@@ -73,7 +73,7 @@ typedef struct clusterLink {
 #define CLUSTER_CANT_FAILOVER_WAITING_DELAY 2
 #define CLUSTER_CANT_FAILOVER_EXPIRED 3
 #define CLUSTER_CANT_FAILOVER_WAITING_VOTES 4
-#define CLUSTER_CANT_FAILOVER_RELOG_PERIOD (60*5) /* seconds. */
+#define CLUSTER_CANT_FAILOVER_RELOG_PERIOD (25) /* seconds. */
```

Comment:
What is the motivation for 25 seconds?

Reply:
Our monitoring mechanism tries to grep logs at a higher frequency than 5 minutes, we suggest the frequency of repeating failed log ranged from 20s to 30s, 25 sec would be a good fit to grep logs.